independent-publisher-connectors/InstagramBasicDisplay/apiDefinition.swagger.json

```diff
@@ -266,6 +266,58 @@
           }
         ]
       }
+    },
+    "/refresh_access_token": {
+        "get": {
+            "responses": {
+                "default": {

```

Comment:
Please check below swagger validation error and fix the same. 

The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1refresh_access_token/get/responses/default

Reply:
I have removed the default value.